src/bizy_server/mw.py

```diff
@@ -0,0 +1,18 @@
+from aiohttp import web
+
+
+# 自定义错误响应
+async def error_middleware(app, handler):
+    async def middleware_handler(request):
+        try:
+            response = await handler(request)
+            return response
+        except web.HTTPRequestEntityTooLarge:
+            return web.json_response(
+                {"error": "File size exceeds the allowed limit. Please use the \"--client_max_size\" option to "
+                          "specify the upload file size limit, default is 10MB."}, status=413
+            )
```

Comment:
```suggestion
            return web.json_response(
                {"error": "File size exceeds the allowed limit(default is 100MB.). Please use the \"--max-upload-size\" option 
 to specify the upload file size limit. For example: python ComfyUI/main.py --max-upload-size 2000"}, status=413
            )
```

我觉得多增加一个示例，告诉用户应该怎么做比较好。另外，这个选项我查了下是 `--max-upload-size`，应该是这个吧，还是说 comfyui 不同版本是不同的？

Reply:
comfyui用client_max_size接收的，这里弄混了。